[Automated] Add w4po.ExplorerTabUtility version 2.4.0

w4po opened the pull request:
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.10 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.10.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/246697)

w4po committed:
[Automated] Add w4po.ExplorerTabUtility version 2.4.0

stephengillie commented:
Automatic Validation ended with:
>  Signature Update Failed. Failure to parse win 10, attempting win 11 Installation failed with exit code -1978335216 2025-04-09 10:16:03.375 [FAIL] WindowsPackageManager.dll!00007FFE995B8212: ReturnHr(1) tid(808) 801901A0 Requested range not satisfiable (416).     Msg:[C:\__w\1\s\external\pkg\src\AppInstallerCommonCore\HttpStream\HttpClientWrapper.cpp(132)\WindowsPackageManager.dll!00007FFE995B0C46: (caller: 00007FFE992EB940) Exception(1) tid(808) 801901A0 Requested range not satisfiable (416). 2025-04-09 10:16:03.375 [FAIL] WindowsPackageManager.dll!00007FFE993472E1: LogHr(1) tid(808) 801901A0 Requested range not satisfiable (416).     Msg:[winrt::hresult_error: Requested range not satisfiable (416).]  2025-04-09 10:16:03.382 [REPO] Source add/update failed, waiting 4451 milliseconds and retrying: winget 2025-04-09 10:16:07.919 [FAIL] WindowsPackageManager.dll!00007FFE995B8212: ReturnHr(2) tid(808) 801901A0 Requested range not satisfiable (416).     Msg:[C:\__w\1\s\external\pkg\src\AppInstallerCommonCore\HttpStream\HttpClientWrapper.cpp(132)\WindowsPackageManager.dll!00007FFE995B0C46: (caller: 00007FFE992EB940) Exception(2) tid(808) 801901A0 Requested range not satisfiable (416). 2025-04-09 10:16:07.919 [FAIL] WindowsPackageManager.dll!00007FFE993472E1: LogHr(3) tid(808) 801901A0 Requested range not satisfiable (416). 2025-04-09 10:16:07.919 [REPO] Failed to update source: winget 2025-04-09 10:16:09.217 [REPO] Failed to open available source: winget 2025-04-09 10:24:05.296 [FAIL] C:\__w\1\s\external\pkg\src\AppInstallerCommonCore\Downloader.cpp(208)\WindowsPackageManager.dll!601DFF4D: (caller: 601E12BD) Exception(1) tid(1db0) 8A15002E  2025-04-09 10:24:05.303 [REPO] Source add/update failed, waiting 7907 milliseconds and retrying: winget 2025-04-09 10:24:13.628 [FAIL] C:\__w\1\s\external\pkg\src\AppInstallerCommonCore\Downloader.cpp(208)\WindowsPackageManager.dll!601DFF4D: (caller: 601E12BD) Exception(2) tid(1db0) 8A15002E  2025-04-09 10:24:13.629 [REPO] Failed to update source: winget 2025-04-09 10:24:13.882 [REPO] Failed to open available source: winget
 
 | Hex | Dec | Inverted Dec | Symbol | Description | 
 | --- | --- | --- | --- | --- | 
 | 0000000B | 11 | -4294967285 | ERROR_BAD_FORMAT | An attempt was made to load a program with an incorrect format. | 
 | 8A150010 | -1978335216 | 2316632080 | APPINSTALLER_CLI_ERROR_NO_APPLICABLE_INSTALLER | None of the installers are applicable for the current system | 
 | 8A15002E | -1978335186 | 2316632110 | APPINSTALLER_CLI_ERROR_DOWNLOAD_SIZE_MISMATCH | Download size does not match expected content length | 


(Automated response - build 1042.)

w4po commented:
Hello @stephengillie 
I hope you are doing great today,

If you kindly please manually check the request, that would be awesome.

Thank you for your work as always. ♥️

w4po commented:
> Hello @@w4po,
> 
> The package manager bot determined there was an issue with installing the application correctly. Please check the application installs correctly. Once repaired, please push an update to your pull request.
> 
> Template: msftbot/validationError/installation/general

It works just fine, needs manual review

stephengillie commented:
Automatic Validation ended with:
>  Signature Update Failed. Failure to parse win 10, attempting win 11 Installation failed with exit code -1978335216 2025-04-09 14:16:10.033 [FAIL] C:\__w\1\s\external\pkg\src\AppInstallerCommonCore\Downloader.cpp(208)\WindowsPackageManager.dll!00007FF92B223F34: (caller: 00007FF92B225749) Exception(1) tid(1dd8) 8A15002E  2025-04-09 14:16:10.042 [REPO] Source add/update failed, waiting 2590 milliseconds and retrying: winget 2025-04-09 14:16:13.275 [FAIL] C:\__w\1\s\external\pkg\src\AppInstallerCommonCore\Downloader.cpp(208)\WindowsPackageManager.dll!00007FF92B223F34: (caller: 00007FF92B225749) Exception(2) tid(1dd8) 8A15002E  2025-04-09 14:16:13.276 [REPO] Failed to update source: winget 2025-04-09 14:16:13.411 [REPO] Failed to open available source: winget 2025-04-09 08:51:26.576 [FAIL] C:\__w\1\s\external\pkg\src\AppInstallerCommonCore\Downloader.cpp(208)\WindowsPackageManager.dll!5F08FF4D: (caller: 5F0912BD) Exception(1) tid(790) 8A15002E  2025-04-09 08:51:26.582 [REPO] Source add/update failed, waiting 5922 milliseconds and retrying: winget 2025-04-09 08:51:32.736 [FAIL] C:\__w\1\s\external\pkg\src\AppInstallerCommonCore\Downloader.cpp(208)\WindowsPackageManager.dll!5F08FF4D: (caller: 5F0912BD) Exception(2) tid(790) 8A15002E  2025-04-09 08:51:32.737 [REPO] Failed to update source: winget 2025-04-09 08:51:32.825 [REPO] Failed to open available source: winget
 
 | Hex | Dec | Inverted Dec | Symbol | Description | 
 | --- | --- | --- | --- | --- | 
 | 0000000B | 11 | -4294967285 | ERROR_BAD_FORMAT | An attempt was made to load a program with an incorrect format. | 
 | 00001722 | 5922 | -4294961374 | ERROR_CLUSTER_PARTIAL_WRITE | An error occurred while writing to a stream of data. The required number of bytes could not be written. | 
 | 8A150010 | -1978335216 | 2316632080 | APPINSTALLER_CLI_ERROR_NO_APPLICABLE_INSTALLER | None of the installers are applicable for the current system | 
 | 8A15002E | -1978335186 | 2316632110 | APPINSTALLER_CLI_ERROR_DOWNLOAD_SIZE_MISMATCH | Download size does not match expected content length | 


(Automated response - build 1042.)

ItzLevvie commented:
@wingetbot run

w4po commented:
@stephengillie

stephengillie commented:
Automatic Validation ended with:
>  Executable C:\Program Files\WiX Toolset v6.0\bin\wix.exe returned exit code: -1 Executable C:\Program Files\WiX Toolset v6.0\bin\wixnative.exe returned exit code: 87 Executable C:\Program Files\WiX Toolset v6.0\bin\x64\burn.exe returned exit code: -2147024883 Executable C:\Program Files\WiX Toolset v6.0\bin\x64\heat.exe returned exit code: 1 Executable C:\Program Files\WiX Toolset v6.0\bin\x86\burn.exe returned exit code: -2147024883 Executable C:\Program Files\WiX Toolset v6.0\bin\x86\heat.exe returned exit code: 1
 
 | Hex | Dec | Inverted Dec | Symbol | Description | 
 | --- | --- | --- | --- | --- | 
 | 00000001 | 1 | -4294967295 | ERROR_INVALID_FUNCTION | Incorrect function. | 
 | 00000057 | 87 | -4294967209 | ERROR_INVALID_PARAMETER | The parameter is incorrect. | 
 | 8007000D | -2147024883 | 2147942413 | Error_invalid_data |  | 
 | FFFFFFFF | -1 | 4294967295 | Cancel |  | 


(Automated response - build 1045.)

stephengillie commented:
Manual Validation ended with:
> 
> 10:19:12 PM:  finished with exit code: -1978335216                                                                      10:19:25 PM: Install Failed.                               
 
 | Hex | Dec | Inverted Dec | Symbol | Description | 
 | --- | --- | --- | --- | --- | 
 | 8A150010 | -1978335216 | 2316632080 | APPINSTALLER_CLI_ERROR_NO_APPLICABLE_INSTALLER | None of the installers are applicable for the current system | 

 
 | Hex | Dec | Inverted Dec | Symbol | Description | 
 | --- | --- | --- | --- | --- | 
 | 8A150010 | -1978335216 | 2316632080 | APPINSTALLER_CLI_ERROR_NO_APPLICABLE_INSTALLER | None of the installers are applicable for the current system | 
 | 8A150010 | -1978335216 | 2316632080 | APPINSTALLER_CLI_ERROR_NO_APPLICABLE_INSTALLER | None of the installers are applicable for the current system | 
 | 8A150010 | -1978335216 | 2316632080 | APPINSTALLER_CLI_ERROR_NO_APPLICABLE_INSTALLER | None of the installers are applicable for the current system | 


(Automated response - build 1045.)

w4po committed:
refactor: remove MinimumOSVersion and add requirement in description (winget)

- Remove MinimumOSVersion: 10.0.22621.0 from installer manifest to avoid
  manual review delays in winget-pkgs validation
- Add Windows 11 requirement information to ShortDescription in locale manifest
- Ensures users are informed about OS requirements while streamlining the
  package acceptance process

w4po commented:
refactor:
- Remove MinimumOSVersion: 10.0.22621.0 from installer manifest to avoid
  manual review delays in winget-pkgs validation
- Add Windows 11 requirement information to ShortDescription in locale manifest
